stack.yaml

```diff
@@ -25,13 +25,13 @@ extra-deps:
 - servant-auth-server-0.4.0.0
 
 - git: https://github.com/serokell/auth-data-structures.git
-  commit: f9a0cd66f443d7b0e04a3c8ae8d475f8344b823b
+  commit: 6fb9e2dd1f3d951996b4e69e690a195551c060e2
 
 - git: https://github.com/serokell/direct-sqlite.git
   commit: 333a2b2cf113e115ea2ec450dc706d3f47ee54e4
 
 - git: https://github.com/serokell/blockchain-util.git
-  commit: 4d8e26e7f1f00449082444ebcdf8ecd6bf422843 # dscp/release-alpha
+  commit: f4459159a6f2ddd1fea0452aa92528bff6eb000b
```

Comment:
And this is neither `dscp/release-alpha` nor master. Please specify what branches are you using in comments.

Reply:
and please not just specify, but rely on a _protected_ branch eventually (which is `master` or `dscp/release-alpha` at present)